lbrynet/extras/daemon/Components.py

```diff
@@ -252,23 +251,18 @@ def component(self):
 
     async def get_status(self):
         if self.wallet_manager and self.running:
-            local_height = self.wallet_manager.network.get_local_height()
-            remote_height = self.wallet_manager.network.get_server_height()
+            local_height = self.wallet_manager.ledger.headers.height
             best_hash = self.wallet_manager.get_best_blockhash()
             return {
                 'blocks': max(local_height, 0),
-                'blocks_behind': max(remote_height - local_height, 0),
```

Comment:
why remove this?

Reply:
`get_server_height()` was stubbed out and not actually returning the server height. Now checking the code, the real network does have a `get_server_height()` so this can be added back. I'll do that.